openqa-maintenance.py

```diff
@@ -1029,6 +1026,10 @@ def setup_checker(self):
         return bot
 
 if __name__ == "__main__":
+    requests_log = logging.getLogger("requests.packages.urllib3")
+    requests_log.setLevel(logging.WARNING)
+    requests_log.propagate = False
+
```

Comment:
urllib uses logger but the config file would have to be parsed by the program that has __main__